tests/next_tests/unit_tests/otf_tests/compilation_tests/build_systems_tests/test_compiledb.py

```diff
@@ -9,6 +9,9 @@
 from gt4py.next import config
 from gt4py.next.otf.compilation import build_data, importer
 from gt4py.next.otf.compilation.build_systems import compiledb
+import shutil
+import tempfile
+import pathlib
```

Comment:
We should add the tests folders to the ruff pre-commit hook 
```suggestion
import pathlib
import shutil
import tempfile
```